salt/modules/slsutil.py

```diff
@@ -54,6 +54,27 @@ def merge(obj_a, obj_b, strategy='smart', renderer='yaml', merge_lists=False):
             merge_lists)
 
 
+def merge_all(lst, strategy='smart', renderer='yaml', merge_lists=False):
+    '''
+    Merge a list of objects into each other in order
+
```

Comment:
Can you add some more documentation to this new function? Each arg/kwarg should be documented.

Can you also add a `..versionadded:: Fluorine` directive?